simcon_project/simcon_project/settings.py

```diff
@@ -45,6 +45,7 @@
     # Our apps
```

Comment:
It wouldn't let me comment on it because it's empty but I'm not sure what the embed_video file is. Might have snuck in somewhere